acknowledgements.tex

```diff
@@ -0,0 +1,13 @@
+\section{Acknowledgments}
+Find the appropriate acknowledgements text at 
+[the how-to-acknowledge Google Doc](https://docs.google.com/spreadsheets/d/1VybV0oMPiqpInTwgO0ICq0EC4AL_fGj-zbXGJAhRiVU/edit?usp=sharing)
+This work was made possible by the wonderful people in ARFC. Additionally,
+this work is funded by [my funding source].
+Prof. Huff is supported by the Nuclear Regulatory Commission Faculty
+Development Program (award NRC-HQ-84-14-G-0054 Program B), the Blue Waters
+sustained-petascale computing project supported by the National Science
+Foundation (awards OCI-0725070 and ACI-1238993) and the state of Illinois,
+the DOE ARPA-E MEITNER Program (award DE-AR0000983), the DOE H2@Scale
+Program (award ), and the International Institute for Carbon Neutral Energy
+Research (WPI-I2CNER), sponsored by the Japanese Ministry of Education,
```

Comment:
out of date. Please see the [the how-to-acknowledge Google Doc](https://docs.google.com/spreadsheets/d/1VybV0oMPiqpInTwgO0ICq0EC4AL_fGj-zbXGJAhRiVU/edit?usp=sharing)